src/tests/ftest/util/dmg_utils.py

```diff
@@ -206,7 +206,7 @@ def storage_scan(self, verbose=False):
 
         return data
 
-    def storage_format(self, reformat=False):
+    def storage_format(self, reformat=False, timeout=20):
```

Comment:
Does the new argument need to be added to the `Args:` description in the comment below?

Reply:
I'll fix the description.  Personally I like the explicit timeouts.